floatingtoast/src/main/java/hari/floatingtoast/FloatingToast.java

```diff
@@ -540,7 +540,7 @@ private void initToast(String messageText, final int duration) {
         layoutParams = dialog.getWindow().getAttributes();
         dialog.requestWindowFeature(Window.FEATURE_NO_TITLE);
         dialog.setContentView(R.layout.layout_toast);
-        dialog.getWindow().setLayout(LinearLayout.LayoutParams.MATCH_PARENT, LinearLayout.LayoutParams.WRAP_CONTENT);
+        dialog.getWindow().setLayout(LinearLayout.LayoutParams.WRAP_CONTENT, LinearLayout.LayoutParams.WRAP_CONTENT);
```

Comment:
Can you make the dialog window match_parent, the text is still not aligned for showAtTouchPosition() toasts